projects/demo/src/modules/components/card-large/examples/12/index.html

```diff
@@ -1,101 +1,46 @@
-<div tuiPlatform="web">
```

Comment:
We still need web vs mobile here, it's just a but of specificity that mobile styles are overrun by desktop. We need to bump specificity here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/addon-mobile/styles/common/card-large.less